<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateImage</class>
 <widget class="QDialog" name="CreateImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建画布</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>预设尺寸：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>宽：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="width">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>——</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>高度：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="height">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>背景：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="background">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>白</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>指定色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透明(明亮格子)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透明(深色格子)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透明(白)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透明(黑)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透明(指定色)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>像素大小：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showWidth">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showHeight">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="seletcedColor">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
